index.html

```diff
@@ -41,5 +41,181 @@ <h1>I’m Marcelo
+        <span><img src="/imagenes/project_2.png" alt="project_2"></span>
+        <span><h2>Multi Post Stories</h2></span>
+        <div class="compn-lan">
+            <p>CANOPY</p>
+            <ul>
+                <li>Back End Dev</li>
+                <li>2015</li>
+            </ul>
+        </div>
+        <div class="card-body">
+            <p>A daily selection of privately
+                 personalized reads; no accounts or 
+                 sign-ups required.</p>
+        </div>
+        <div class="technology">
+            <ul>
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+            </ul>
+        </div>
+        <button class="button">See Project</button>
+    </div>  
+</div>
+<!-- project-3 -->
+<div class="projects">
+    <div class="card">
+        <span><img src="/imagenes/project_3.png" alt="project_3"></span>
+        <span><h2>Tonic</h2></span>
+        <div class="compn-lan">
+            <p>CANOPY</p>
+            <ul>
+                <li>Back End Dev</li>
+                <li>2015</li>
+            </ul>
+        </div>
+        <div class="card-body">
+            <p>A daily selection of privately
+                 personalized reads; no accounts or 
+                 sign-ups required.</p>
+        </div>
+        <div class="technology">
+            <ul>
+                <li>html</li>
+                <li>css</li>
+                <li>Java Script</li>
+            </ul>
+        </div>
+        <button class="button">See Project</button>
+    </div>
+</div>
+<!-- project-4 -->
+<div class="projects">
+    <div class="card">
+        <span><img src="/imagenes/project_4.png" alt="project_4"></span>
+        <span><h2>Multi-Post Stories</h2></span>
+        <div class="compn-lan">
+            <p>CANOPY</p>
+            <ul>
+                <li>Back End Dev</li>
+                <li>2015</li>
+            </ul>
+        </div>
+        <div class="card-body">
+            <p>A daily selection of privately
+                 personalized reads; no accounts or 
+                 sign-ups required.</p>
+        </div>
+        <div class="technology">
+            <ul>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Java Script</li>
+            </ul>
+        </div>
+        <button class="button">See Project</button>
+    </div>
+</div>
+</section>
+<!-- About-section -->
+<section class="about-me">
+    <div class="me-content">
+        <span id="span"><h2>About Myself</h2></span>
+        <div class="about-details">
+            <p>Hello I’m a software developer! I can help you 
+                build a product , feature or website Look through 
+                some of my work and experience! If you like what you 
+                see and have a project you need coded, don’t hestiate
+                 to contact me.</p>
+        </div>
+        <div class="connect">
+            <span>LET’S CONNECT</span>
```

Comment:
Good job! but don't you think this should be a link?

Please make sure to replace it with a link tag.